data/human/deep missions.txt

```diff
@@ -4042,10 +4045,10 @@ mission "Hjlod Remembers Windblain"
 		has "Hjlodsaved"
 	on offer
 		dialog
-			`Your inbox hums with a new message from <planet>:`
-			`	Dear <first>,`
+			`"Your inbox hums with a new message from <planet>:`
```

Comment:
Should there be a quotation mark here? This seems like a typo, the quote begins one line below (where there is another opening quotation mark)